packages/fuels-test-helpers/src/lib.rs

```diff
@@ -127,13 +128,24 @@ pub fn setup_single_asset_coins(
     coins
 }
 
+pub fn setup_single_message(owner: &Bech32Address, amount_per_message: u64) -> Vec<Message> {
```

Comment:
It would be nice if this function allowed more parts to be specified for more detailed tests down the line. I made a similar function for testing that looked like this:
``` rust
pub fn setup_single_message(
    sender: Address,
    recipient: Address,
    amount: Word,
    nonce: Word,
    data: Vec<u8>,
) -> Message {
    Message {
        sender: sender,
        recipient,
        owner: recipient,
        nonce,
        amount,
        data: data,
        da_height: DaBlockHeight::default(),
        fuel_block_spend: None,
    }
}
```

Also, you'll notice it uses `recipient` for both the message `recipient` and `owner` to prepare for when `owner` will eventually be dropped in another issue. I do like how you are returning an array though and I think you should keep that.